src/silx/gui/plot/backends/BackendMatplotlib.py

```diff
@@ -941,7 +941,18 @@ def addShape(
         return item
 
     def addMarker(
-        self, x, y, text, color, symbol, linestyle, linewidth, constraint, yaxis, font
+        self,
+        x,
+        y,
+        text,
+        color,
+        symbol,
+        linestyle,
+        linewidth,
+        constraint,
+        yaxis,
+        font,
+        bgcolor: str = None,
```

Comment:
```suggestion
        bgcolor: str | None,
```

No need for default value, none of the backend methods have any.